internal/project/repository/eventsourcing/project.go

```diff
@@ -99,6 +100,21 @@ func projectStateAggregate(aggCreator *es_models.AggregateCreator, project *mode
 	}
 }
 
+func ProjectRemovedAggregate(ctx context.Context, aggCreator *es_models.AggregateCreator, existing *model.Project) (*es_models.Aggregate, error) {
+	if existing == nil {
+		return nil, errors.ThrowPreconditionFailed(nil, "EVENT-Cj7lb", "Errors.Internal")
+	}
+	agg, err := ProjectAggregate(ctx, aggCreator, existing)
+	if err != nil {
+		return nil, err
+	}
+	agg, err = agg.AppendEvent(model.ProjectRemoved, existing)
+	if err != nil {
+		return nil, err
+	}
+	return agg, nil
```

Comment:
we dont need data on a remove event
direct return
```suggestion
	return agg.AppendEvent(model.ProjectRemoved, nil)
```